src/libraries/System.IO.Pipelines/src/System/IO/Pipelines/PipeCompletionCallbacks.cs

```diff
@@ -27,13 +27,10 @@ public void Execute()
 
             List<Exception>? exceptions = null;
 
-            if (_callbacks != null)
```

Comment:
>             [](http://example.com/codeflow?start=0&length=12)

should we add assert for `_callbacks != null` in line 15?

Reply:
We certainly could.  Though it's a readonly field that's initialized via a non-nullable parameter.  And I double checked call sites and the only one guarantees it's not null.  No strong opinion on adding an assert, though if we wanted to add one it'd be better to add it in the ctor I think.

Reply:
no strong opinion either. I am ok either way you prefer.